sdcm/logcollector.py

```diff
@@ -384,33 +383,21 @@ def get_grafana_annotations(self, grafana_ip: str) -> str:
         return ""
 
     @staticmethod
-    def dashboard_exists(grafana_ip, uid):
-        """Check on Grafana server, that dashboard exists
-
-        Send request to Grafana and validate that dashboard with uid
-        provided as "uid" parameter is available and could be requested
-
-        Arguments:
-            node {CollectingNode} -- Remote host with grafana server
-            uid {str} -- uid of grafana dashboard
+    def search_dashboard(grafana_ip: str, port: int, query: str) -> list:
```

Comment:
Let's make it retry couple of times before failure, please take into account that requests.get can raise

Reply:
Done